boards/px4/fmu-v4/init/rc.board_mavlink

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+#
+# PX4 FMUv4 specific board MAVLink startup script.
+#------------------------------------------------------------------------------
+
+# Start MAVLink on the USB port
+mavlink start -d /dev/ttyACM0
```

Comment:
This is done automatically now based on VBUS.

Reply:
Ok, will remove